arts-university-bournemouth.csl

```diff
@@ -0,0 +1,254 @@
+  <info>
+    <title>Arts University Bournemouth</title>
+    <title-short>AUB</title-short>
+    <id>http://www.zotero.org/styles/arts-university-bournemouth</id>
+    <link href="http://www.zotero.org/styles/arts-university-bournemouth" rel="self"/>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <link href="http://www.zotero.org/styles/the-open-university-harvard" rel="template"/>
+    <link href="http://www.aublibrary.info/index.php/referencing" rel="documentation"/>
+    <author>
+      <name>Duncan Belk</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <updated>2017-04-24T10:20:30+00:00</updated>
+  </info>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="text" initialize-with=". " delimiter=", " sort-separator=", " name-as-sort-order="all"/>
+      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+ <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" and="text" sort-separator=", " delimiter=", " delimiter-precedes-last="never" form="long" initialize-with=". "/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <choose>
+          <if type="webpage">
+            <text variable="container-title"/>
+            <text variable="URL"/>
+          </if>
+          <else>
+            <names variable="editor"/>
+            <text macro="anon"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter=", " delimiter-precedes-last="never" initialize-with=". "/>
+      <substitute>
+        <choose>
+          <if type="webpage">
+            <text variable="container-title"/>
+            <text variable="URL"/>
+          </if>
+          <else>
+            <names variable="editor"/>
+            <names variable="translator"/>
+            <text macro="anon"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="translator">
+    <names variable="translator">
+      <label form="short" plural="always" suffix=" "/>
+      <name and="symbol" delimiter=", " delimiter-precedes-last="never" initialize-with=". "/>
+    </names>
+  </macro>
+  <macro name="director">
+    <names variable="author" prefix="Directed by "/>
+  </macro>
+  <macro name="access">
+    <group delimiter=" ">
+      <text term="online" prefix="[" suffix="]." text-case="capitalize-first"/>
+      <choose>
+        <if variable="DOI">
+          <text variable="DOI" prefix="DOI: "/>
```

Comment:
Since this is your institution's style -- any chance you'd reconsider how DOIs are displayed? This format is no longer recommended by CrossRef (as well as the DOI foundation and other DOI agencies):
https://www.crossref.org/display-guidelines/
Instead, use a full link, i.e. something like `<text variable="DOI" prefix="https://doi.org/"/>`